packages/dashboard/src/components/Wizard/PanelWizard.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { IPanelWizardProps, IPanelWizardStyles, IPanelWizardStyleProps } from './PanelWizard.types';
+import { styled, IRenderFunction, classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { Panel, PanelType, IPanelProps } from 'office-ui-fabric-react/lib/Panel';
+import { Wizard } from './Wizard';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+import { defaultPanelStyleSet, defaultWizardStyleSet, getPanelWizardStyles } from './PanelWizard.styles';
+import { getStepContentToShow } from './Wizard.utils';
+
+const getClassNames = classNamesFunction<IPanelWizardStyleProps, IPanelWizardStyles>();
+
+/** Component for Setup Wizard */
+class PanelWizardBase extends React.Component<IPanelWizardProps, {}> {
+  public render(): JSX.Element {
+    if (this.props.wizardProps.steps.length === 0) {
+      throw new Error('Wizard must have atleast one step.');
+    }
+
+    const panelStyle = mergeStyleSets(
+      defaultPanelStyleSet(this.props.theme!),
+      this.props.panelProps ? this.props.panelProps.styles : undefined
+    );
+    const wizardStyle = mergeStyleSets(
+      defaultWizardStyleSet(this.props.theme!),
+      this.props.wizardProps.styles ? this.props.wizardProps.styles : undefined
+    );
+
+    return (
+      <Panel

```

Comment:
>Panel [](start = 7, length = 5)

Could you also remove the empty files from the PR?  PanelWizardAction*

Reply:
will do.